apps/sensenet/src/components/AddButton.tsx

```diff
@@ -254,7 +254,7 @@ export const AddButton: React.FunctionComponent<AddButtonProps> = (props) => {
                       queryParams: { 'content-type': childType.ContentTypeName },
                     })
                   }}
-                  data-test="listitem">
+                  data-test={`${childType.DisplayName}`}>
```

Comment:
You can change the data-test attribute of an element, if no other test using it. In this case the attribute is already used in AddNewMenu.spec.ts (twice: line30 and line45). Please update this test also. Thanks!